pkg/clusteragent/admission/mutate/auto_instrumentation_test.go

```diff
@@ -401,6 +401,78 @@ func TestExtractLibInfo(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:              "all",
+			pod:               fakePodWithAnnotation("admission.datadoghq.com/all-lib.version", "latest"),
+			containerRegistry: "registry",
+			expectedLibsToInject: []libInfo{
```

Comment:
maybe add a comment here for futur languages: `TODO: add new entry when a new language is supported`

Reply:
Done!